changelogs/fragments/24.yaml

```
@@ -0,0 +1,5 @@
+---
+minor_changes:
+  - Add a ``store_yaml_stream`` function to ``antsibull_core.yaml`` to dump
+    YAML to an IO stream.
```

You seem to like these periods before the final brackets :)
```suggestion
    YAML to an IO stream
```

---

It seems I do :)